### PR TITLE
Nt province and cities select

### PR DESCRIPTION
**Pull request by nooralnabi**

### CODE REVIEW

File: app/controllers/ads_controller.rb [hunk 1]
```diff
@@ -86,10 +94,12 @@ def ad_reach
         end
       end
     end
+    p "************@customer_by_id_state*****"
```
Comment (ssharba) [inline app/controllers/ads_controller.rb]: remove this lines

File: app/models/address.rb [hunk 2]
```diff
@@ -16,14 +16,18 @@ def address
 
 
   def self.customer_by_id(province,city,gender,start_date,end_date)
-    city_name = City::Translation.where(id:city).pluck(:name)
+    p province
+    p "****province****"
```
Comment (ssharba) [inline app/models/address.rb]: this too

File: app/controllers/ads_controller.rb (hunk 1 above)
Comment (nooralnabi) [reply app/controllers/ads_controller.rb]: ok

File: app/models/address.rb (hunk 2 above)
Comment (nooralnabi) [reply app/models/address.rb]: ok

**Pull request merged**